### ISSUE TITLE
Can't use Resources, Images

### ISSUE THREAD

**Issue by Hyperatox** [LegacyModdingMC/examplemod#2]
### Description
Can't use images or other resources in forge mod dev environment.

### Environment I tested it with:
openjdk version "1.8.0_382"
OpenJDK Runtime Environment (Zulu 8.72.0.17-CA-win64) (build 1.8.0_382-b05)
OpenJDK 64-Bit Server VM (Zulu 8.72.0.17-CA-win64) (build 25.382-b05, mixed mode)
and Oracle Java 8 
 

### To Reproduce:
1. Clone [this test repo](https://github.com/Hyperatox/MissingImages)
OR
1. Clone a clean template.
2. Try to use any Resources, within the resource location().
3. Task runClient

### To fix the issue, I tried:

-  on a different Computer with oracle java 8.
- on a different IDE: VSCode, IntelliJ IDEA Community Edition
- with a different image.
- checked for differences to the original example mod with win merge.
- adding 2 lines to the manifest:
-
			"FMLCorePluginContainsFMLMod": true,
			"ForceLoadAsMod"		  :	true,
                        **this fixed the issue in non dev environments!**

- 2 weeks of Experimenting with the runtimeClasspath, client args, jvmArgs.
Can be found commanded out ([here](https://github.com/Hyperatox/MissingImages/blob/1d6d9a4d0e5ab38084608cea1564c3cb5f4cdcfa/build.gradle#L153))

### Expected behavior:
be able to use resources in dev env.
or (for my example project) To load the Ruby Texture using mixin on the Main Menu.

No success, I need a hero...

**Comment by wagyourtail** [comment]
I know why this is happening (at least on the fabric one) but I'm not quite sure how I want to fix it... it's an issue with how the gradle's set up. I consider this multi-project style to be the "deprecated" way to use unimined, and only want to support it for ease of porting things like architectury mods to it.

**Comment by Hyperatox** [comment]
### Another way from @wagyourtail on Discord

slight buildscript change should now fix it if you are on the newest 1.2.0-SNAPSHOT snapshot.
after [this line](https://github.com/LegacyModdingMC/examplemod/blob/e33f84c0fff8cee4b5a2a7db50594c9052aa6d1e/build.gradle#L139) add
```
if (project.name !== "Common") {
    combineWith(":Common:main")
}
```

then run gradlew --refresh-dependencies so it re-makes the mc jar with the new patches that I added to forge to support it 
lmk if you encounter any issues with that, the fix is quite new and hasn't been fully tested yet
also, as of the 1.2.0 snapshots,
common could be using accessWidener instead of legacyFabric as its patcher
please note, the 1.2.0 snapshots are latest, where I am allowed to break things (and consequently do bigger fixes like this), if you want a more stable experience 1.1.2 is the latest LTS version. tho I may just move 1.2.0 to lts soon and just move onto 1.3 since more people are now using it
